Parsers/ASimWebSession/Parsers/ASimWebSession.yaml

```diff
@@ -1,63 +1,65 @@
-Parser:
```

Comment:
@jaimeesc Please make sure only the new change shows. Currently, its difficult to validate the change, as whole file shows changed.

Reply:
I'm not sure why this is happening. I'll try rolling it back.

Reply:
I rolled back the file, updated to current master state, then updated again with my changes. Please let me know if it looks okay this time. These changes look fine on my end when I check my commits.

Reply:
I'm not sure what I can do with this file. Here's a diff of the file compared to the master branch. I just added a comma to the end of the previous line, then added a line for vimWebSessionSonicWallFirewall.

![image](https://github.com/Azure/Azure-Sentinel/assets/42151366/9e92a3c2-5f14-4ad5-b0dc-a37071342434)

![image](https://github.com/Azure/Azure-Sentinel/assets/42151366/de1f316a-6626-4818-969e-6568e6071453)